packages/desktop-client/src/components/LoggedInUser.js

```diff
@@ -27,14 +28,16 @@ function LoggedInUser({
 }) {
   let [loading, setLoading] = useState(true);
   let [menuOpen, setMenuOpen] = useState(false);
+  const url = useServerURL();
```

Comment:
nitpick: variable naming could be improved to be more precise
```suggestion
  const serverUrl = useServerURL();
```